samples/msgext-copilot-handoff/ts/src/index.ts

```diff
@@ -0,0 +1,121 @@
+  {},
+  credentialsFactory
+);
+
+const adapter = new CloudAdapter(botFrameworkAuthentication);
+
+// Catch-all for errors.
+const onTurnErrorHandler = async (context: TurnContext, error: Error) => {
+  // This check writes out errors to console log .vs. app insights.
+  // NOTE: In production environment, you should consider logging this to Azure
+  //       application insights.
+  console.error(`\n [onTurnError] unhandled error: ${error}`);
+
+  // Send a trace activity, which will be displayed in Bot Framework Emulator
+  await context.sendTraceActivity(
+    "OnTurnError Trace",
+    `${error}`,
+    "https://www.botframework.com/schemas/error",
+    "TurnError"
+  );
+
+  // Send a message to the user
+  await context.sendActivity(`The bot encountered unhandled error:\n ${error.message}`);
+  await context.sendActivity("To continue to run this bot, please fix the bot source code.");
+};
+
+// Set the onTurnError for the singleton CloudAdapter.
+adapter.onTurnError = onTurnErrorHandler;
+
+// Create the bot that will handle incoming messages.
+// const conversationReferences = {};
+const continuationParameters: {} = {};
+const searchApp = new SearchApp(async () => {
+  console.log(
+    `Handling continuation - ${JSON.stringify(continuationParameters)}`
+  );
+  for (const continuationParameter of Object.values(continuationParameters)) {
+    const conversationReference = (continuationParameter as any)
+      .conversationReference;
+
+      await adapter.continueConversationAsync(
+        (continuationParameter as any).claimsIdentity,
+        conversationReference,
+        (continuationParameter as any).oAuthScope,
+        async (context) => {
+          // MicrosoftAppCredentials.trustServiceUrl(
+          //   conversationReference.serviceUrl
+          // );
+          const continuationToken = (continuationParameter as any)
+            .continuationToken;
+          await context.sendActivities([
+            {
+              type: ActivityTypes.Message,
+              text: "Continuing conversation from copilot...",
+            },
+            { type: ActivityTypes.Typing },
+            { type: "delay", value: 1000 },
+            {
+              type: ActivityTypes.Message,
+              text: `Fetching more details using the continuation token passed: ${continuationToken}`,
+            },
+            { type: ActivityTypes.Typing },
+            { type: "delay", value: 4000 },
+            {
+              type: ActivityTypes.Message,
+              text: `Handoff successful!`,
+              attachments: [(continuationParameter as any).cardAttachment],
+            },
+            { type: ActivityTypes.Typing },
+            { type: "delay", value: 2000 },
+            {
+              type: ActivityTypes.Message,
+              text: `Do you need revenue or discounts details about ${(
+                continuationToken as string
+              ).replace("-continuation", "")}?`,
+            },
```

Comment:
remove this part. Let's stop the flow after the success card is shown.

Reply:
removed